test/e2e/tree_test.go

```diff
@@ -0,0 +1,89 @@
+package integration
+
+import (
+	"fmt"
+	"os"
+
+	. "github.com/containers/libpod/test/utils"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var treeOutput = `Image ID: 68fd96c9b58f
+Tags    : [docker.io/library/wordpress:php7.2-fpm]
```

Comment:
The tests depend on an image that we do not control, which can cause issues, for instance, when `wordpress:php7.2-fpm]` gets removed or changed. Personally, I am okay with just simple smoke tests without comparing the output.

Reply:
Other test cases also use images from dockerhub. 
Do you mean, I should not compare the output and maybe check for some part of output?

Reply:
> Other test cases also use images from dockerhub.

The other tests do not compare the output with hardcoded IDs, sizes, etc. The current tests are non-deterministic in the sense that input can vary at any time. If 
wordpress:php7.2-fpm gets updated, the tests would break as the IDs etc. are likely to change.

> Do you mean, I should not compare the output and maybe check for some part of output?

Personally, I would be okay to just run the commands and make sure that the exit code is 0. Comparing the output is prone to errors unless we control the image. @baude, do we have some images on Quay that we could reuse here?

Reply:
output comparison replaced by exit status check